src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -413,6 +439,7 @@ public void getProjectByUnknownTagTest() {
     @Test
     public void createProjectTest(){
         Project project = new Project();
+        project.setAccessTeams(new ArrayList<Team>());
```

Comment:
Making `accessTeams` a mandatory field would be a breaking change in the API. Initializing it with an empty list should not be necessary. The resource code should expect this field to be missing (`null`), and handle it accordingly.